eks-distro-base/create_pr.sh

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bash
+# Copyright 2020 Amazon.com Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+set -e
+set -o pipefail
+set -x
+
+REPO="$1"
+OLD_TAG="$2"
+NEW_TAG="$3"
+FILEPATH="$4"
+
+if [ $REPO = "eks-distro-build-tooling" ]; then
+    FILENAME="Tag file"
+elif [ $REPO = "eks-distro" ]; then
+    FILENAME="Makefile"
+else
+    FILENAME="EKS Distro base periodic"
+fi
```

Comment:
I am confused by the filenames here. What are the referring to, not actual filenames and rather a description or something?